docs/data-cookbooks/setup.md

```diff
@@ -1,12 +1,12 @@
 ---
 id: setup
-title: Data Cookbooks Setup
+title: Datacookbooks Introduction
```

Comment:
🐜 
```suggestion
title: Data Cookbooks Introduction
```

Reply:
Er.... how'd *that* happen.   Nice catch :|